sw/device/lib/testing/otbn_testutils.h

```diff
@@ -150,17 +151,19 @@ typedef uint32_t otbn_addr_t;
  *
  * @param otbn The context object.
  * @param app The application to load into OTBN.
+ * @return The result of the operation.
  */
-void otbn_testutils_load_app(const dif_otbn_t *otbn, const otbn_app_t app);
+status_t otbn_testutils_load_app(const dif_otbn_t *otbn, const otbn_app_t app);
```

Comment:
nit: may be a bit safer with a warning if the result isn't checked.
```suggestion
OT_WARN_UNUSED_RESULT
status_t otbn_testutils_load_app(const dif_otbn_t *otbn, const otbn_app_t app);
```
Same for the other functions here.

Reply:
Make sense, done.